frontend/packages/console-app/console-extensions.json

```diff
@@ -163,31 +167,51 @@
         "version": "v1",
         "kind": "OAuth"
       },
+<<<<<<< HEAD
       "component": { "$codeRef": "oauthConfigDetailsPage" }
+=======
+      "component": {
+        "$codeRef": "oauthConfigDetailsPage.default"
+      }
+>>>>>>> 4d8d84e63c (Adds CRUD support for VolumeGroupSnapshot)
```

Comment:
merge conflicts were not properly resolved...

Reply:
Sorry about that, something is wrong with the editor. It somehow stopped highlighting the conflicts